database/src/main/java/com/firebase/ui/database/FirebaseRecyclerAdapter.java

```diff
@@ -31,9 +31,9 @@
         extends RecyclerView.Adapter<VH> implements FirebaseAdapter<T> {
     private static final String TAG = "FirebaseRecyclerAdapter";
 
-    protected ObservableSnapshotArray<T> mSnapshots;
-    protected Class<VH> mViewHolderClass;
-    protected int mModelLayout;
+    protected final ObservableSnapshotArray<T> mSnapshots;
```

Comment:
@puf Were you saying we should make a `protected ObservableSnapshotArray<T> getSnapshots()` method and make this field private?